apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/ErrorReporting/ErrorReportingExceptionLoggerExtension.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Microsoft.AspNetCore.Builder;
+
+namespace Google.Cloud.Diagnostics.AspNetCore
+{
+    /// <summary>
+    ///  Uses the Google Cloud Error Reporting Exception Logger Middleware.
+    ///  To ensure all unhandled exceptions are reported, this should be
+    ///  the first piece of middleware used.
+    /// </summary>
+    /// 
+    /// <example>
+    /// <code>
+    /// public void Configure(IApplicationBuilder app)
+    /// {
+    ///     // Use first To ensure all unhandled exceptions are reported.
+    ///     string projectId = "[Google Cloud Platform project ID]";
+    ///     string serviceName = "[Name of service]";
+    ///     string version = "[Version of service]";
+    ///     app.ReportExceptionsToGoogle(projectId, serviceName, version);
+    ///     ...
+    /// }
+    /// </code>
+    /// </example>
```

Comment:
An extensions class lacks a bit of discoverability. It might make sense to also add this example to `ErrorReportingExceptionLogger`.